python/Dockerfile

```diff
@@ -43,7 +43,8 @@ RUN apt-get update \
     libxml2-dev \
     libxmlsec1-dev \
     libgeos-dev \
-    python3-enchant
+    python3-enchant \
+    libkrb5-dev
```

Comment:
@jeffwidman Thank you so much for the review! Can I please ask if we are waiting for other admins to review? 

Also, would you want the change you mentioned in this PR itself? Or in a future PR?

Reply:
> If we were to do this, it'd be better to break it into a separate section of "deps required not for our python setup, but for user packages that have C-deps."

Agreed - ideally we would only install these "extra" packages after attempting an update and encountering a build error.

I think we have been struggling to update tensorflow for similar reasons so we should think about this deeper given that tensorflow vulnerabilities are [relatively common](https://github.com/advisories?query=tensorflow)

I do think this change in particular is ~fine. libkrb5 [has builds for our system](https://packages.debian.org/sid/libkrb5-dev) so we don't have to build from source, and the actual library is 150kb.

For something like glibc, though, we would want to be careful since it could increase our update run times and we might run into versioning/linking issues depending on the project we're trying to update.

Only requested change here is to install it as a separate RUN command so the Dockerfile can continue building even if the install command fails.

```
RUN apt-get install -y \
    libkrb5-dev \
    ; exit 0
```

Reply:
made the changes recommended by @jeffwidman. Thank you for all the help and reviews :)